faw/pdf-observatory/common/processor.ts

```diff
@@ -42,164 +42,215 @@ export function reprocess(
   // Build sets of filter groups
   const filesWithMessages = Object.entries(reprocessPdfGroups.groups);
 
+  function evalCheck(message: string, check: any, identifier: string): Set<number> {
```

Comment:
No change to this function's internals; it was just moved outside of the scope below, and `identifier` was made a parameter.